src/main/js/controls/Line/LineConfig.js

```diff
@@ -54,6 +54,8 @@ class LineConfig extends BaseConfig {
      */
     clone() {
         this.config = utils.deepClone(this.input);
+        this.config.strokeDasharray =
```

Comment:
Do you have a better way of doing this?

Reply:
There are two more ways I can think of:
1. Use this condition directly where this property is being used. But, I don't think, that is the way it has been developed in carbon-graph
2. Instead of creating a property strokeDasharray, update the style property object to have default value, if not present.
`this.config.style = {
strokeDasharray: (this.config.style && this.config.style.strokeDasharray) || "0";
}`
and then use `this.config.style.strokeDasharray` wherever necessary. With this approach, if there are any more property that is being added to the style, we can update in the same code.

Reply:
Cant you add it here: https://github.com/cerner/carbon-graphs/blob/master/src/main/js/controls/Line/helpers/helpers.js#L275

Reply:
I have fixed this and updated the label casing as well.